### PR TITLE
update build script

### PR DESCRIPTION
**Pull request by xlc**

### CODE REVIEW

**Review by zjb0807 — COMMENTED** [review]
LGTM.
```
evm-bench
evm-tests
ts-tests
```
This needed to be added in `.dockerignore`

File: Makefile [hunk 1]
```diff
@@ -32,19 +32,19 @@ build-all:
 
 .PHONY: build-release
 build-release:
-	CARGO_PROFILE_RELEASE_LTO=true RUSTFLAGS="-C codegen-units=1" cargo build --locked --features with-all-runtime --release --workspace --exclude runtime-integration-tests --exclude e2e-tests
+	CARGO_PROFILE_RELEASE_LTO=true RUSTFLAGS="-C codegen-units=1" cargo build --locked --features with-all-runtime --release --workspace --exclude runtime-integration-tests --exclude e2e-tests --exclude test-service
```
Comment (zqhxuyuan) [inline Makefile:35]: test-service is the only child package inside e2e-tests, I guess we could only exclude test-service, but I think its fine to also exlude e2e-tests, as we may be add other e2e test case later on.

Comment (xlc) [reply Makefile:35]: I noticed that test-service was also compiled if we don't exclude it.

Comment (zqhxuyuan) [reply Makefile:35]: yeah, test-service needs excluded.

**Review by zjb0807 — APPROVED** [review]

**Pull request merged**